lib/linter/linter.js

```diff
@@ -510,6 +510,7 @@ function normalizeVerifyOptions(providedOptions, config) {
 
     return {
         filename: normalizeFilename(providedOptions.filename || "<input>"),
+        physicalFilename: normalizeFilename(providedOptions.physicalFilename || "<input>"),
```

Comment:
We probably want this here:
```suggestion
        physicalFilename: normalizeFilename(providedOptions.filename || "<input>"),
```